fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -324,7 +324,6 @@ private void dispatch() throws IOException {
             return;
         }
         ctx.setCommand(command);
-        ctx.setStartTime();
```

Comment:
Other stmt also need to set startime. So I think we can't move this method here?